README.md

```diff
@@ -172,6 +172,17 @@ import * as openpgp from './openpgp.min.mjs';
 
 To offload cryptographic operations off the main thread, you can implement a Web Worker in your application and load OpenPGP.js from there. For an example Worker implementation, see `test/worker/worker_example.js`.
 
+#### TypeScript
+
+For typescript support you have to add `@openpgp/web-stream-tools` to your dependencies as well.
+
+```sh
+npm install --save openpgp
```

Comment:
```suggestion
Since TS is not fully integrated in the library, TS-only dependencies are currently listed as `devDependencies`, so to compile the project you’ll need to add `@openpgp/web-stream-tools` manually:

```sh
```

Whether OpenPGP.js is to be installed as dev or not depends on the user's stack. so I'd leave that one out.